package.json

```diff
@@ -75,7 +76,8 @@
     "gulp": "^3.9.1",
     "lerna": "^2.5.1",
     "lint-staged": "^5.0.0",
-    "prettier": "^1.8.2"
+    "prettier": "^1.8.2",
+    "typescript": "^3.3.4000"
```

Comment:
Could this one be `^3.3.1`? That way we ensure that we have a single typescript version in the whole repo.

Reply:
Fixed